### DISCUSSION TITLE
Error searching messages, API responded with status 400! 	{"message":"Invalid Form Body","code":50035,"errors":{"max_id":{"_errors":[{"code":"NUMBER_TYPE_COERCE","message":"Value \"tengu.playz.games@gmail.com\" is not snowflake."}]}}} CoreException	{}

### DISCUSSION THREAD

**Discussion by CombatCalamity** [victornpb/undiscord#589] [2. Discussions]
HELP ME PLS IT JUST SAYS Error searching messages, API responded with status 400!
	{"message":"Invalid Form Body","code":50035,"errors":{"max_id":{"_errors":[{"code":"NUMBER_TYPE_COERCE","message":"Value \"tengu.playz.games@gmail.com\" is not snowflake."}]}}}
CoreException	{} WHEN I CLICK DELETE

**Comment by SD-Chargers** [comment]
Nice eMail

Anyhow. You're using Chrome ( or any other Chrome based webbrowser?)

I had the same problem, found that the application was using my stored "password manager" infos as API tokens.

Deleted all saved password in the browser for discord.com, and - suprise - it worked again.

Try that and lemme know

**Reply by yeditemmuz** [reply]
it is working now, thank you

**Comment by MatthewDMerrill** [comment]
I had this issue too, I just used incognito mode and had to enable tampermonkey extension to be used in incognito mode, but it worked fine.
